package/MDAnalysis/coordinates/base.py

```diff
@@ -1741,7 +1749,66 @@ def get_aux_descriptions(self, auxnames=None):
             auxnames = self.aux_list
         descriptions = [self._auxs[aux].get_description() for aux in auxnames]
         return descriptions
-
+    
+    @property
+    def transformations(self):
+        """ Returns the list of transformations"""
+        return self._transformations
+    
+    @transformations.setter
+    def transformations(self, transformations):
+        if not self._transformations:
+            self._transformations = transformations
+        else:
+            raise ValueError("Transformations are already set")
+        
+    def add_transformations(self, *transformations):
+        """ Add all transformations to be applied to the trajectory.
+        
+        This function take as list of transformations as an argument. These
+        transformations are functions that will be called by the Reader and given
+        a :class:`Timestep` object as argument, which will be transformed and returned
+        to the Reader.
+        The transformations can be part of the :mod:`~MDAnalysis.transformations` 
+        module, or created by the user, and are stored as a list `transformations`. 
+        This list can only be modified once, and further calls of this function will
+        raise an exception.
+                           
+            u = MDAnalysis.Universe(topology, coordinates)
```

Comment:
This bloc of code should be in an explicit code block, it should star with `.. code:: python` to avoid sphinx to choke on `*workflow` as it does now.: <http://www.sphinx-doc.org/en/stable/markup/code.html>.

The sphinx error to prevent is:

    /home/travis/build/MDAnalysis/mdanalysis/package/MDAnalysis/coordinates/DCD.py:docstring of MDAnalysis.coordinates.DCD.DCDReader.add_transformations:15:Inline emphasis start-string without end-string.